docs: Add ns2 docs (#3963)
kyma-project/busola: docs/features.md

Note that some features must be run before the application starts the bootstrap process, so they are out of the normal feature flow.

> #### Features list for Web

Author: ```suggestion
#### Features list for Frontend
```

    isEnabled: true
  ```

## Features List for Backend

> [!NOTE]
> Backend features cannot be modified at the cluster's ConfigMap level.

- **GZIP** – is used to indicate whether a response from the backend server should be compressed or not.

Default settings:

```yaml
GZIP:
  isEnabled: true
```

> - **KYMA_COMPANION** - is used to configure location of Kyma companion API.

Reviewer: ```suggestion
- **KYMA_COMPANION** - is used to configure the location of the Kyma companion API.
```

    isEnabled: true
    link: https://www.youtube.com/watch?v=dQw4w9WgXcQ
  ```

- **EXTENSIBILITY_INJECTIONS**

> - **EXTENSIBILITY_WIZARD**

Author: ```suggestion
```
I spoke with Mateusz, and we're not adding this.

> - **KYMA_COMPANION** - determines if the kyma companion chat window is available in busola

Author: ```suggestion
- **KYMA_COMPANION** - determines if the Kyma Companion chat window is available in Busola
```

  Default settings:

  ```yaml

>   FEEDBACK:

Author: ```suggestion
  KYMA_COMPANION:
```

    isEnabled: false
  ```

> - **SNOW** - determines if snow feature is enabled in busola

Author: ```suggestion
- **SNOW** - determines if the snow animation is enabled in Busola
```
